zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -168,6 +168,62 @@ sealed trait Http[-R, +E, -A, +B] { self =>
   def unwrap[R1 <: R, E1 >: E, C](implicit ev: B <:< ZIO[R1, E1, C]): Http[R1, E1, A, C] =
     self.flatMap(Http.fromEffect(_))
 
+  /**
+   * Returns an Http that peeks at the success of this Http.
+   */
+  def tap[R1 <: R, E1 >: E, A1 <: A](f: B => Http[R1, E1, Any, Any]): Http[R1, E1, A, B] =
+    self.flatMap(v => f(v).as(v))
+
+  /**
+   * Returns an Http that effectfully peeks at the success of this Http.
+   */
+  def tapM[R1 <: R, E1 >: E](f: B => ZIO[R1, E1, Any]): Http[R1, E1, A, B] =
```

Comment:
Should we name these `tapEffect`? Or just skip the current `tap*` and rename these to `tap*` instead?

Reply:
This is consistent with ZIO.

Reply:
👍